acceptance/tests/puppetserver_requests.rb

```diff
@@ -0,0 +1,26 @@
+require 'beaker-http'
+
+test_name 'Ensure that requests can be built out to the puppetserver'
+
+step 'install latest released puppet agent' do
+  install_puppet_agent_on(hosts)
+end
+
+step 'install the latest puppet-server on the master' do
+  install_package(master, 'puppetserver')
+  on master, 'service puppetserver start'
+end
+
+step 'generate a new beaker http connection object'
+beaker_http_connection = Beaker::Http::Connection.new(master)
+
+step 'configure the connection to connect to the master port'
+beaker_http_connection.connection.port = 8140
```

Comment:
This seems like the kind of thing we're trying to avoid through hiding (touching the connection under the connection), which makes it awkward to say, and binds us to Faraday's implementation. 

Couldn't this be added to the `def_delegators` line in the connection, so that this could be
```ruby
beaker_http_connection.port  = 8140
```